nixos/modules/config/resolvconf.nix

```diff
@@ -54,6 +54,14 @@ in
         '';
       };
 
+      package = mkOption {
+        type = types.package;
+        internal = true;
```

Comment:
Not sure why this is internal.

Reply:
No good reason.

Reply:
Well, I think my intention was that users shouldn't see this in the manual and think they have to set it. Maybe just say that in the description?

Reply:
I don't know. A good description and default value can get you a long way.

Reply:
I extended the description and set `openresolv` as the default.